bagel/tests/data/README.md

```diff
@@ -18,4 +18,11 @@ Example inputs to the CLI
 | synthetic    | valid, has `participant` and `session` IDs corresponding to the [`synthetic` example BIDS dataset](https://github.com/bids-standard/bids-examples/tree/master/synthetic) | valid                                                                                | pass               |
 | 11           | invalid, ex 6 with missing entries in `participant_id` and `session_id` columns                                                                                                           | valid, based on example 6                                                            | fail               |
 
-`* this is expected to fail until we enable multiple participant_ID handling`.
\ No newline at end of file
+`* this is expected to fail until we enable multiple participant_ID handling`.
+
+## Example expected CLI outputs
+
+| Example                             | Description                                                                                                                                                                                                                                                                                                                         |
+| ----------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| example_synthetic.jsonld            | Sample output of `bagel pheno` command on the `synthetic` example inputs. Contains subject-level annotated phenotypic attributes obtained by essentially applying the data dictionary (.json) to the original tabular data (.tsv).                                                                                                  |
```

Comment:
No change requested here for this PR, but a reminder to use more descriptive example names since these two examples could be confused and don't fully describe their contents.

Here's that issue you were mentioning on the topic: https://github.com/neurobagel/bagel-cli/issues/72